lte/gateway/c/oai/tasks/gtpv1-u/gtp_tunnel_openflow.c

```diff
@@ -387,5 +389,12 @@ const struct gtp_tunnel_ops* gtp_tunnel_ops_init_openflow(void) {
     openflow_multi_tunnel_init();
   }
 
+  // If pipeline config is enabled initialize userplane ops
+  if (spgw_config.sgw_config.ovs_config.pipelined_managed_tbl0) {
+    OAILOG_INFO(LOG_GTPV1U, "Using upf classifier for gtp apps");
+
+    return &upf_openflow_ops;
```

Comment:
can you move this to the UPF file.

Reply:
Comments addressed. Wrote a wrapper function in UPF file.

Reply:
Updated the patch based on latest comments.